tools/XCConfigExporter/Fixtures.bundle/iOSApp/iOSApp.xcodeproj/project.pbxproj

```diff
@@ -200,38 +175,13 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = YES;
```

Comment:
This probably shouldn't be here either, and this tool is only used for diagnostic settings